src/main/java/by/a1qa/screens/DiscoverScreen.java

```diff
@@ -0,0 +1,54 @@
+package by.a1qa.screens;
+
+import aquality.appium.mobile.actions.SwipeDirection;
+import aquality.appium.mobile.elements.ElementType;
+import aquality.appium.mobile.elements.interfaces.ILabel;
+import aquality.appium.mobile.screens.Screen;
+import by.a1qa.models.Show;
+import by.a1qa.utils.PriceParser;
+import org.openqa.selenium.By;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DiscoverScreen extends Screen {
+    private final String showContainer = "//android.widget.LinearLayout[@index='1']";
+    private final String xpathContainer = "//android.view.ViewGroup[@content-desc='Fan Favourites The shows that our audience love']";
```

Comment:
what is `xpathContainer`?

Reply:
So since we have some different containers with identical id's(show_name, price_amount_label), i have to point on concrete container where i'm searching for my elements

Reply:
name is awful